packages/perseus-core/src/widgets/sorter/sorter-util.ts

```diff
@@ -17,13 +20,23 @@ type SorterPublicWidgetOptions = {
 function getSorterPublicWidgetOptions(
     options: PerseusSorterWidgetOptions,
 ): SorterPublicWidgetOptions {
+    const correctToString = options.correct.join("");
+    const hashes = CoreUtil.getHashes(correctToString);
+    const getSeed = CoreUtil.getSeedFunctionFromHashes(...hashes);
+    const seedFromCorrect = getSeed();
+
+    const shuffledCorrect = CoreUtil.shuffle(
+        options.correct,
+        seedFromCorrect,
+        /* ensurePermuted */ true,
+    );
```

Comment:
Looking at this, I think it could be simplified. `CoreUtil.shuffle` has its own seeded random number generator, so we might not need the sfc32 algorithm.

```suggestion
    const [seed] = CoreUtil.getHashes(correctToString);

    const shuffledCorrect = CoreUtil.shuffle(
        options.correct,
        seed,
        /* ensurePermuted */ true,
    );
```

Reply:
So then this would just grab the first number of the hash and use that in the shuffle? Does that decrease the security level of the hash? Is there a greater chance that it will be the same as a hash for another problem?